crates/bevy_text/src/text.rs

```diff
@@ -105,3 +113,22 @@ impl Default for TextStyle {
 pub struct Text2dSize {
     pub size: Size,
 }
+
+#[cfg(feature = "bevy_default_assets")]
+pub(crate) mod default_font {
+    use crate::{Font, DEFAULT_FONT_HANDLE};
+    use ab_glyph::{FontArc, FontRef};
+    use bevy_app::AppBuilder;
+    use bevy_asset::Assets;
+
+    pub(crate) fn load_default_font(app: &mut AppBuilder) {
+        let world = app.world_mut();
+        let world_cell = world.cell();
+        let mut fonts = world_cell.get_resource_mut::<Assets<Font>>().unwrap();
+        let font_bytes = include_bytes!("assets/fonts/FiraSans-Bold.ttf");
```

Comment:
This adds ~500KB to the size of the `bevy_text` package (currently lib.rs thinks it is 1.5MB, so an additional 1/3).

In theory, we should have this in a different crate, however because the amount of metadata needed for creating a crate anyway, it might not be worthwhile, if 95% of people leave the default font in.

Reply:
maybe a default assets crate? if other fonts / binary assets get added, could go in the same one?